contracts/ListaStakeManager.sol

```diff
@@ -367,28 +356,8 @@ contract ListaStakeManager is
 
         uint256 coveredAmount = 0;
         uint256 coveredSlisBnbAmount = 0;
-        uint256 oldLastUUID = withdrawalQueue.length != 0 ? withdrawalQueue[0].uuid - 1 : requestUUID;
-
-        // old requests will be fully covered by the last undelegated() call, can be removed in next version
-        for (uint256 i = nextConfirmedRequestUUID; i <= oldLastUUID; ++i) {
-            BotUndelegateRequest storage botRequest = uuidToBotUndelegateRequestMap[i];
-            if (undelegatedQuota < botRequest.amount) {
-                totalDelegated -= coveredAmount;
-                if (coveredSlisBnbAmount > 0) {
-                    ISLisBNB(slisBnb).burn(address(this), coveredSlisBnbAmount);
-                }
-                emit ClaimUndelegatedFrom(_validator, nextConfirmedRequestUUID, undelegatedAmount);
-                return (nextConfirmedRequestUUID, undelegatedAmount);
-            }
-            botRequest.endTime = block.timestamp;
-            undelegatedQuota -= botRequest.amount;
-            coveredAmount += botRequest.amount;
-            coveredSlisBnbAmount += botRequest.amountInSnBnb;
-            ++nextConfirmedRequestUUID;
-        }
 
-        // new logic, new requests exist; `withdrawalQueue[0].uuid <= nextConfirmedRequestUUID` condition can be removed in next version
-        if (withdrawalQueue.length != 0 && withdrawalQueue[withdrawalQueue.length - 1].uuid >= nextConfirmedRequestUUID && withdrawalQueue[0].uuid <= nextConfirmedRequestUUID) {
+        if (withdrawalQueue.length != 0 && withdrawalQueue[0].uuid <= nextConfirmedRequestUUID) {
```

Comment:
double check if should use 
`withdrawalQueue.length != 0 && withdrawalQueue[withdrawalQueue.length - 1].uuid >= nextConfirmedRequestUUID`